### ISSUE TITLE
Windows builds missing

### ISSUE THREAD

**Issue by Anthchirp** [conda-forge/bitshuffle-feedstock#18]
Would be nice to have though

**Comment by xmnlab** [comment]
yes, that would be great! I don't remember very well the history here but probably I tried to include windows support but as it didn't work I just removed that at that time to move forward this package to the feedstock :)

**Comment by Anthchirp** [comment]
I just had a look and ... agree with your assessment.

**Comment by dagewa** [comment]
I installed a Windows build of bitshuffle from https://anaconda.org/paulscherrerinstitute/bitshuffle, which seemed to work. I was limited to trying it on [Python 3.7 though](https://github.com/paulscherrerinstitute/conda-recipes/issues/19). I'm not familiar enough with conda builds to know what is different in their case and whether Windows builds could be brought here by following those steps.

**Comment by xmnlab** [comment]
@dagewa thanks for the information! 

if you want to check the recipe used by that package you can:

1. download the tar file https://anaconda.org/paulscherrerinstitute/bitshuffle/0.3.6.1/download/win-64/bitshuffle-0.3.6.1-py37he980bc4_0.tar.bz2

2. extract the file

3. check the files at `info/recipe`

with just an overview I didn't find any big issue to add support for windows.

could you open a new PR just removing the skip item? https://github.com/conda-forge/bitshuffle-feedstock/blob/master/recipe/meta.yaml#L17

maybe now it would work :) if not, I can try to help you there :)
